packages/pluggableWidgets/maps-native/src/Maps.editorConfig.ts

```diff
@@ -0,0 +1,20 @@
+import { Problem } from "@mendix/piw-utils-internal";
+import { MapsPreviewProps } from "../typings/MapsProps";
+
+export function check(values: MapsPreviewProps): Problem[] {
+    const errors: Problem[] = [];
+    const zoomLevels = ["world", "continent", "country", "city", "town", "streets", "building"];
+
+    const minZoomLevelIndex = zoomLevels.indexOf(values.minZoomLevel);
+    const maxZoomLevelIndex = zoomLevels.indexOf(values.maxZoomLevel);
+
+    if (minZoomLevelIndex > maxZoomLevelIndex) {
+        errors.push({
+            property: "minZoomLevel",
+            severity: "error",
+            message: "The minimum zoom level can not be smaller than the maximum zoom level."
```

Comment:
I think you want to make the checks both ways:

1: `Min zoom level cannot be bigger than the maximum zoom level`
2: `Max zoom level cannot be smaller than the minimum zoom level`

Reply:
good one, updated